dashboard/test/controllers/registrations_controller_test.rb

```diff
@@ -156,6 +156,14 @@ class RegistrationsControllerTest < ActionController::TestCase
     assert_equal ["Email has already been taken"], assigns(:user).errors.full_messages
   end
 
+  test "create causes UserGeo creation" do
+    assert_creates(UserGeo) do
+      post :create, params: {
+        user: @default_params.merge({current_sign_in_at: '1.2.3.4'})
```

Comment:
the `current_sign_in_at` param isn't doing anything in this test (`user: @default_params` is sufficient).
The default request ip in this test is `0.0.0.0`, to specify/verify an explicit IP set `request.remote_addr = '1.2.3.4'` (then verify that the created `UserGeo` contains the same IP).